src/utilities.h

```diff
@@ -630,16 +630,16 @@ std::string makeUniqueId(IdList &idList);
  * Function to support linking units names to their corresponding @c Units items.
```

Comment:
@ref Units, not @c.

Reply:
See https://github.com/cellml/libcellml/pull/836#discussion_r634778487.

Reply:
Done in commit b904f47.